### PR TITLE
feat(sequencer-removal): remove sequencer code

### LINKED ISSUE #667: Remove sequencer code and related modules from linea-arithmetization project

### PR DESCRIPTION
**Pull request by powerslider**
Resolves: #667 
Signed-off-by: Tsvetan Dimitrov <tsvetan.dimitrov@consensys.net>

### CODE REVIEW

File: arithmetization/build.gradle [hunk 1]
```diff
@@ -47,7 +47,6 @@ dependencies {
   implementation "${besuArtifactGroup}.internal:core:${besuVersion}"
   implementation "${besuArtifactGroup}.internal:rlp:${besuVersion}"
   implementation "${besuArtifactGroup}.internal:algorithms:${besuVersion}"
-  implementation project(":native:compress")
```
Comment (OlivierBBB) [inline arithmetization/build.gradle:50]: Is that the bane of our existence ?

File: .github/workflows/gradle.yml [hunk 2]
```diff
@@ -39,66 +39,6 @@ jobs:
...
-      - name: Checkout repository
-        uses: actions/checkout@v3
-
-      - name: Set up JDK 17
-        uses: actions/setup-java@v3
-        with:
-          java-version: '17'
-          distribution: 'adopt'
-
-      - name: Run libcompress JNI tests
```
Comment (OlivierBBB) [inline .github/workflows/gradle.yml:90]: Is that the culprit ?

**Review by OlivierBBB — COMMENTED** [review]
LGTM

**Review by Gabriel-Trintinalia — APPROVED** [review]
LGTM. is zkevm-constraints updated modeule part of this change?

**Pull request merged**